content/c-sharp/concepts/Conditionals/conditionals.md

```diff
@@ -0,0 +1,152 @@
+---
+Title: 'conditionals'
+Description: 'Compares inputs and gives an boolean output depending on it being True or False.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Dev'
+  - 'App Dev'
+  - 'Game Dev'
+Tags:
+  -
+CatalogContent:
+  - 'Learn C#'
+---
+
+## Explanation
+
+Below we'll follow the `if`, `else` and, `else if` statements one by one while expanding the example.
+
+## if statements
+
+An `if statement` compares two values and returns a value `if` true.
+
+## Example 1
+```cs
+
+var input1 = 10;
+var input2 = 10;
+var output;
+
+if (input1 === input2){
+    output = true; // sets the value of output to true.
+}
+```
+
+If the code above did not return true the code below will return false.
+
+```cs
+if(output === true){
+    Console.WriteLine('I returned true')
+}
+
+```
+>**note** `===` means equal. `!==` means not equal. 
+
+## Explanation of example 1
+
+We start by creating three variables (`var`) and assigning a value to two of them. The third variable will be assigned in a later stage of the code.
+
+After creating the variables we start our `if statement`.
+
+`if` input1 is equal to input 2 then set the variable of output to true. In our example, input1 was equal to input 2. The `if statement` check was true so it ran the code setting the value of output to true. 
+
+after that, we wrote a second `if statement` which checked if the output was equal to true. The output of this was true so it ran the code and wrote *I returned true* to the console. if the first `if statement` would have been false. The output would not get assigned a value and the second `if statement` would return false meaning no text will be written to the console.
+
+## else statements
+
+A `else statement` goes in combination with the `if statement` and returns a value `if` false.
+
+## Example 2
```

Comment:
There should be a separation between different elements in the entry (headings, code blocks, notes, etc.)